### ISSUE TITLE
heketi-cli volume create fails with no space

### ISSUE THREAD

**Issue by sofo110111e** [heketi/heketi#505]
Hello,

While running command below I am getting "Error: No space"

heketi-cli volume create --size=1 --clusters=7b377075af001357f34c748df02d1ebc --persistent-volume-endpoint=10.0.2.15 --name=/mnt/brick1/newvol2

...according to journalctl 

Sep 20 13:58:08 localhost.localdomain heketi[722]: [heketi] INFO 2016/09/20 13:58:08 Creating volume 58db37cd13885f8a06021c062a373fd0
Sep 20 13:58:08 localhost.localdomain heketi[722]: [heketi] DEBUG 2016/09/20 13:58:08 /src/github.com/heketi/heketi/apps/glusterfs/volume_entry.go:255: Using the following clusters: []
Sep 20 13:58:08 localhost.localdomain heketi[722]: [heketi] ERROR 2016/09/20 13:58:08 /src/github.com/heketi/heketi/apps/glusterfs/volume_entry_allocate.go:41: Minimum brick size limit reached.  Out of space.
Sep 20 13:58:08 localhost.localdomain heketi[722]: [negroni] Started GET /queue/70d38b6384926ccac46fdc265261ff63
Sep 20 13:58:08 localhost.localdomain heketi[722]: [negroni] Completed 200 OK in 61.609µs
Sep 20 13:58:08 localhost.localdomain heketi[722]: [heketi] ERROR 2016/09/20 13:58:08 /src/github.com/heketi/heketi/apps/glusterfs/app_volume.go:131: Failed to create volume: No space

Can you help me to resolve this issue.
I am running glusterfs as a pod and connected to that via Kubernetes executor

**Comment by lpabon** [comment]
Sorry, we should probably have a better error.  You cannot make volumes of less than 10 G at the moment.  We are working on adding future block storage support for smaller storage needs.  See #481 .

**Comment by lpabon** [comment]
I will create an issue to show that the size of the volume is too small, the error can be better.  See #506

**Comment by lpabon** [comment]
One more thing, again I know it is bad message and needs to be fixed, but the error is here:

```
Sep 20 13:58:08 localhost.localdomain heketi[722]: [heketi] ERROR 2016/09/20 13:58:08 /src/github.com/heketi/heketi/apps/glusterfs/volume_entry_allocate.go:41: Minimum brick size limit reached. Out of space.
```

**Comment by lpabon** [comment]
Reopen to track #506

**Comment by lpabon** [comment]
This issue still needs #527

**Comment by lpabon** [comment]
Closed by #544

**Comment by Vishal-Gaur** [comment]
hiii @sofo110111e
can you tell me how i can see the heketi logs using journalctl ...????
